stdlib/boolean/boolean.flux

```diff
@@ -0,0 +1,9 @@
+// Package boolean provides constants for true and false values.
```

Comment:
Can you change the boolean package to something like `internal/vars`? I'd like to avoid adding a new package that confuses users to believe it is something other than an internal detail. This package is only needed to resolve a dependency cycle in the prelude for the standard library and should never be imported manually. The internal signifies that this should not be used directly. I like `vars` as the base name because it signifies we can put other builtin variables in here too.

Reply:
I move the package to `internal/boolean`. I  chose the name `boolean` so its clear that no other values should be added to the package. Since we are trying to avoid any dependencies within the boolean package, because any of those package will not be able to import the boolean package. This way its clear nothing more should be added to this boolean package.